src/forms/OrgForm/OrgForm.js

```diff
@@ -14,30 +27,119 @@ const orgSchema = yup.object().shape({
   bio: yup.string()
 });
 
-const OrgForm = ({ onSubmit, defaultValues = {}, validate = false }) => (
+// eslint-disable-next-line react/prop-types
+const AvatarEdit = ({ initialURL, imageUploadHandler = () => {} }) => {
+  const fileRef = useRef();
+  const [avatarSrc, setSrc] = useState(initialURL);
+  const { notifyDispatch = () => console.log("Notify") } = useNotify();
```

Comment:
is this intended?

Reply:
Yes - It needs to be own component so it can update form state